src/operators/subgraph.c

```diff
@@ -359,6 +359,57 @@ igraph_error_t igraph_i_induced_subgraph_map(const igraph_t *graph, igraph_t *re
     }
 }
 
+/**
+ * \ingroup structural
+ * \function igraph_induced_subgraph_map
+ * \brief Creates an induced subraph and returns the mapping from the original.
+ *
+ * </para><para>
+ * This function collects the specified vertices and all edges between
+ * them to a new graph.
+ * As the vertex IDs in a graph always start with zero, this function
+ * very likely needs to reassign IDs to the vertices.
+ * \param graph The graph object.
```

Comment:
```suggestion
 * This function collects the specified vertices and all edges between
 * them to a new graph.
 * As the vertex IDs in a graph always start with zero, this function
 * very likely needs to reassign IDs to the vertices.
 *
 * \param graph The graph object.
```

The `<para>` doesn't hurt, it's just unnecessary. The empty line before the `\param` is for readability in code.